lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -555,6 +559,7 @@ instance DefineSeverity StakePoolLog where
         MsgStakeDistribution _ -> Info
         MsgStakePoolRegistration _ -> Info
         MsgRollingBackTo _ -> Info
+        MsgUsingRankingEpochConstants _ -> Info
```

Comment:
This ought to be `Debug`, no ?

Reply:
Hm… you're probably right

Reply:
I wonder if it would be wrong to increase it to `Notice` only when `R == 0` and `Debug` otherwise.

Reply:
I didn't do that. I think it would be neat, but I want to notice when the stake is 0 too. So I did add two new NOTICE traces: ff3bf9b373778dfd1e9f57c33f61e18d57a46c45 for when `totalStake` and `totalRewards` are 0

Reply:
Good idea (modulo the comment about where to put the logic regarding the interpretation of logging). 
Still, this one should really be Debug.

Reply:
now debug